src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -2629,6 +2629,9 @@ bool CacheManager::Init(const std::string& file_name,
       LOG4CXX_DEBUG(logger_,
                     "Check if snapshot valid: " << std::boolalpha << result);
 
+      if (!UnwrapAppPolicies(pt_->policy_table.app_policies_section.apps)) {
+        LOG4CXX_ERROR(logger_, "Cannot unwrap application policies");
+      }
```

Comment:
Please add a newline after this if block.

Reply:
Newline was added in commit 9ddff2ba34b270929df979e63ef50043e363490a